include/ccf/node_context.h

```diff
@@ -3,19 +3,21 @@
 #pragma once
 
 #include "ccf/historical_queries_interface.h"
+#include "ccf/host_processes_interface.h"
 #include "ccf/indexing/indexer_interface.h"
 #include "ccf/indexing/lfs_interface.h"
-#include "node/rpc/node_interface.h"
```

Comment:
Here's the goal - 406 additions and 130 deletions for the privilege of deleting _exactly this line_.